tests/Shared/ScopedSanityCheck.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Avalonia.UnitTests;
+using Xunit;
+
+namespace Avalonia.UnitTests.Helpers;
+
+static class ScopedSanityCheck
+{
+    [ModuleInitializer]
```

Comment:
`VerifyEmptyDispatcherAfterTestAttribute` could probably be deleted now.

Reply:
We can probably utilize it for tests that aren't expected to leave jobs in dispatcher.

As of Loaded queue, it should be redone completely with OnLoaded virtual method being removed. It's currently very inconsistent with WPF too.